commands/mmctl_test.go

```diff
@@ -91,3 +91,11 @@ func (s *MmctlE2ETestSuite) RunForAllClients(testName string, fn func(client.Cli
 		fn(s.th.LocalClient)
 	})
 }
+
+// RunUnprivilegededClient runs a test using unauthenticated client
```

Comment:
typo

Reply:
Oops. Let me fix it and update PR.

Reply:
Hey @akshaychhajed - how is it going with this one? :)

Reply:
I received comments for my enable plugin PR which may be relevant to this pull request too.

https://github.com/mattermost/mmctl/pull/274#discussion_r518806300

I wanted to test it with unprivileged user not in local mode.
I will need to review the different modes for the client to get a better picture.